pkg/controller/master/master_controller.go

```diff
@@ -160,31 +221,31 @@ func (c *MasterController) Run(workers int, stopCh <-chan struct{}) {
 	<-stopCh
 }
 
-func (c *MasterController) enqueue(machine *clusteroperator.Machine) {
-	key, err := controller.KeyFunc(machine)
+func (c *MasterController) enqueue(machineSet *clusteroperator.MachineSet) {
+	key, err := controller.KeyFunc(machineSet)
 	if err != nil {
-		utilruntime.HandleError(fmt.Errorf("Couldn't get key for object %#v: %v", machine, err))
+		utilruntime.HandleError(fmt.Errorf("Couldn't get key for object %#v: %v", machineSet, err))
 		return
 	}
 
 	c.queue.Add(key)
 }
 
-func (c *MasterController) enqueueRateLimited(machine *clusteroperator.Machine) {
-	key, err := controller.KeyFunc(machine)
+func (c *MasterController) enqueueRateLimited(machineSet *clusteroperator.MachineSet) {
+	key, err := controller.KeyFunc(machineSet)
 	if err != nil {
-		utilruntime.HandleError(fmt.Errorf("Couldn't get key for object %#v: %v", machine, err))
+		utilruntime.HandleError(fmt.Errorf("Couldn't get key for object %#v: %v", machineSet, err))
 		return
 	}
 
 	c.queue.AddRateLimited(key)
 }
 
-// enqueueAfter will enqueue a machine after the provided amount of time.
-func (c *MasterController) enqueueAfter(machine *clusteroperator.Machine, after time.Duration) {
-	key, err := controller.KeyFunc(machine)
+// enqueueAfter will enqueue a machine set after the provided amount of time.
+func (c *MasterController) enqueueAfter(machineSet *clusteroperator.MachineSet, after time.Duration) {
```

Comment:
Is there a case where we foresee using this?

Reply:
Yeah, I think this is what we would use when we run the period reconciliation that re-asserts that everything is set up in AWS.

Reply:
sgtm